circle.yml

```diff
@@ -44,6 +46,8 @@ deployment:
       - TESTPILOT_BUCKET=testpilot.dev.mozaws.net ./bin/deploy.sh dev
       - aws configure set preview.cloudfront true
       - aws cloudfront create-invalidation --distribution-id E2ERG47PHCWD0Z --paths '/*'
+      - docker pull owasp/zap2docker-weekly
+      - timeout 300 docker run -t owasp/zap2docker-weekly zap-baseline.py -t https://testpilot.dev.mozaws.net/
```

Comment:
Is there a better place this could go then? Or do we not get to know when its actually been deployed?